src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -115,4 +129,57 @@ public void nullValue() {
 
     Assert.assertEquals(actual, expected);
   }
+
+  @Test
```

Comment:
nit: Add comment before this test.
`/** Tests for getUsersToNotify() function */`

Reply:
Done!